Sources/TuistKit/GraphMappers/ProjectMapperProvider.swift

```diff
@@ -53,6 +48,11 @@ final class ProjectMapperProvider: ProjectMapperProviding {
         // Support for resources in libraries
         mappers.append(ResourcesProjectMapper())
 
+        // Auto-generation of schemes
+        if !config.generationOptions.contains(.disableAutogeneratedSchemes) {
```

Comment:
This is worth adding a unit test [here](https://github.com/tuist/tuist/blob/main/Tests/TuistKitTests/GraphMappers/ProjectMapperProviderTests.swift) to ensure that the mapper is in the right position.

Reply:
Also, what about adding a comment explaining why it has to be in this position?